client/src/app/site/projector/models/view-projector.ts

```diff
@@ -48,6 +48,10 @@ export class ViewProjector extends BaseViewModel {
         return this.projector ? this.projector.scroll : null;
     }
 
+    public get reference_projector_id(): number {
```

Comment:
doc

Reply:
We don't have docs there at all, like all simple getters in all view models. For these getters it's ok..